runtime-manager/modules/ROOT/pages/rtm-traffic-inspection.adoc

```diff
@@ -0,0 +1,111 @@
++
+[NOTE] 
+The Mule runtime installation bundle includes both the Mule Runtime Engine and the Mule Runtime Agent. 
+
+=== Upgrade the Mule Runtime Agent
+
+Make sure that the version of the Mule Runtime agent is `2.5.6` or later. If you have an earlier version, update the runtime agent before registering.
+
+To update the Runtime agent:
+
+. Download the agent's zip file
+. Extract the downloaded `agent-setup-2.5.6.zip` file to `$MULE_HOME/bin`.
++
+If prompted, overwrite any conflicting files.
+. Do NOT run `amc_setup -U`.
+
+=== Register the Mule Runtime
+
+. Update the file `wrapper.conf` file with the IP and port of the traffic inspection proxy by following instructions in xref:rtm-agent-proxy-config.adoc#set-up-proxy-server-configuration-in-the-wrapper-conf-file[Set Up Proxy Server Configuration in the wrapper.conf File].
+. Login in to the Anypoint console.
+. From the Anypoint Platform, select *Runtime Manager*.
+. Click *Servers* in the navigation menu.
+. Click *Add Server* .
++
+image::traffic-add-server.png[]
++
+. In a terminal window, change the `$MULE_HOME/bin` directory to the Mule instance that you're registering.
+. Paste the command on the command line and append the proxy's IP address or domain name and port, and the `--enable-traffic-inspection` configuration flag. 
++
+[source,console,linenums]
+----
+./amc_setup -H {registrationToken} {serverName} -P {proxy ip or hostname} {proxyPort} --enable-traffic-inspection
+----
++
+[NOTE]
+Make sure to leave a space between the proxy's domain name and port number.
++
+. Confirm that the mule has registered successfully. The runtime should show up as *Created* in the Anypoint console:
++
+image::mule-registered.png[]
++
+. Edit the file `$MULE_HOME/conf/mule-agent.yml` and set the property `authenticationProxy.endpoint` to `null`.
+. Start the mMule.
+
+=== Check that the Mule Runtime is Connected to the Control Plane
+
+If the connection is successful, you should see the runtime with status *Running* in the Anypoint console:
+
+image::mule-running.png[]
+
+Also, if the connection has been established, the Mule agent terminal window displays the following message:
+
+[source,console,linenums]
+----
+INFO  2023-04-19 17:27:41,307 [WebSocketInboundExecutor] [processor: ; event: ] com.mulesoft.agent.transport.handlers.GenericWebSocketHandler: Opening Mule Agent WebSocket
+INFO  2023-04-19 17:27:41,316 [WebSocketInboundExecutor] [processor: ; event: ] com.mulesoft.agent.transport.handlers.GenericWebSocketHandler: Mule Agent WebSocket opened
+INFO  2023-04-19 17:27:41,316 [pool-12-thread-1] [processor: ; event: ] com.mulesoft.agent.transport.connections.AsyncHttpWSConnectionThread: Mule Agent WebSocket connection was initialized after: 1 attempts
+INFO  2023-04-19 17:27:42,179 [WebSocketInboundExecutor] [processor: ; event: ] com.mulesoft.agent.services.security.HandshakeAuthorizationService: WebSocket Client connection authorized
+----
+
+== Notes
```

Comment:
This seems like a catch-all section. Is there any way to include this information elsewhere? Maybe in the intro? In general, we want to avoid headings like "Notes" and put important things earlier in the topic, with descriptive headings.